.github/CODEOWNERS

```
@@ -6,8 +6,8 @@
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-/**                      @rickwinter @LarryOsterman
-/sdk/                    @rickwinter @LarryOsterman
+/**                      @rickwinter @LarryOsterman @ahsonkhan @antkmsft @gearama
+/sdk/                    @rickwinter @LarryOsterman @ahsonkhan @antkmsft @gearama
```

This should be the lead. It is rare it gets hit, When it does, it means the owners list for libraries is not properly configured.

 ```suggestion
/sdk/                    @rickwinter
```

---

Makes sense. Updated

---

> This should be the lead. It is rare it gets hit, When it does, it means the owners list for libraries is not properly configured.

Looks like we hit this case whenever a new service and SDK is introduced for the first time, which means a new folder gets added under `/sdk/`. Like AppConfig, most recently.

Do we want to keep the lead as the only/sole approver for such cases?

---

See https://github.com/Azure/azure-sdk-for-cpp/pull/6200 as an example of this